mitodl/hugo-course-publisher
# tweaks for "course section" mobile view

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #85

#### What's this PR do?

this is a PR that implements the top-level sections, like the course syllabus, basically just making a few styling tweaks to get things in line.

I also made a tweak pulling out the instructor name and course number into a partial for both the home page and the course sections so we can style it in one place (`course_instructor_number.html`).

#### How should this be manually tested?

check out the designs on invision, check out a bunch of pages, make sure things look alright.

#### Screenshots (if appropriate)

![Screenshot from 2020-02-26 15-41-16](https://user-images.githubusercontent.com/6207644/75386881-6fb76600-58b0-11ea-848e-440f0c443582.png)
![Screenshot from 2020-02-26 15-41-08](https://user-images.githubusercontent.com/6207644/75386883-704ffc80-58b0-11ea-9ae9-b34f1053aaae.png)
